src/providers/seventv/SeventvEventAPI.cpp

```diff
@@ -261,4 +331,53 @@ void SeventvEventAPI::onUserUpdate(const Dispatch &dispatch)
     }
 }
 
+// NOLINTBEGIN(readability-convert-member-functions-to-static)
+
+// We're using `Application::instance`, because we're not in the GUI thread.
+// `seventvBadges` does its own locking.
```

Comment:
Move this comment to inside each function where Application::instance is used

Maybe something like

```c++
// We're using `Application::instance` instead of getApp(), because we're not in the GUI thread.
// `seventvBadges` does its own locking.
auto *seventvBadges = Application::instance->seventvBadges;
...
```

include `getApp()` in the comment to make it easier to find